docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -361,47 +358,52 @@ mysql> flush privileges;
 Installation on CentOS 7
 ------------------------
 
-First make sure that you system is correctly configured and up-to-date
+Step 1: Install the replicated database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+NOTE: In this example, the database is replicated in active/passive across two servers that are part of the PacketFence cluster. Active/active replication is also possible using MariaDB Galera cluster but this subject is not covered in this guide.
+
+
+Creation of the DRBD partition
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During the OS installation, reduce the size of the main partition and create a new one (that will be used for the replicated MySQL database) of 30G. *Do not* create that partition during the install process, we will do it later. This needs to be done on both servers.
```

Comment:
Partitioning section begins at line 375. That section was moved.

at 375:
"After installation, you need to create the extra partition for DRBD. Using fdisk, create your new partition and save the table. You will probably need to reboot your server after this step. This also needs to be done on both servers."